src/core_read.cpp

```diff
@@ -44,10 +45,17 @@ opcodetype ParseOpCode(const std::string& s)
             }
         }
     }
+    opcodetype Parse(const std::string& s) const {
```

Comment:
nit: according to clang-format:

```diff
diff --git a/src/core_read.cpp b/src/core_read.cpp
index bd29e2073e..ba535f745f 100644
--- a/src/core_read.cpp
+++ b/src/core_read.cpp
@@ -45,7 +45,8 @@ public:
             }
         }
     }
-    opcodetype Parse(const std::string& s) const {
+    opcodetype Parse(const std::string& s) const
+    {
         auto it = mapOpNames.find(s);
         if (it == mapOpNames.end()) throw std::runtime_error("script parse error: unknown opcode");
         return it->second;

Reply:
Nit coding-style: opening brace for functions/classes on a separate line.